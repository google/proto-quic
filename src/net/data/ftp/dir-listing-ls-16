dr-xr-xr-x   1 owner    group               0 Nov 28  2008 documentales
dr-xr-xr-x   1 owner    group               0 Nov 28  2008 dosier
dr-xr-xr-x   1 owner    group               0 Dec  1  2008 promos
dr-xr-xr-x   1 owner    group               0 Nov 28  2008 Sueños_futbol
dr-xr-xr-x   1 owner    group               0 Nov  2 15:53 test
dr-xr-xr-x   1 owner    group               0 Nov 25 10:04 tmp
-r-xr-xr-x   1 owner    group             125 Oct 11  2007 Gastronomía.txt
