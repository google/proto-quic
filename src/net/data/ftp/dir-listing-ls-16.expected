d
documentales
-1
2008
11
28
0
0

d
dosier
-1
2008
11
28
0
0

d
promos
-1
2008
12
1
0
0

d
Sueños_futbol
-1
2008
11
28
0
0

d
test
-1
1994
11
2
15
53

d
tmp
-1
1993
11
25
10
4

-
Gastronomía.txt
125
2007
10
11
0
0
